docker/Dockerfile

```diff
@@ -85,7 +85,7 @@ RUN \
 RUN python -m pip install --no-cache-dir torchserve torch-model-archiver torch-workflow-archiver
 
 # Final image for production
-FROM ${BASE_IMAGE} AS runtime-image
+FROM ${BASE_IMAGE} AS production-image
```

Comment:
do you mean nvidia production image? if so could you add a comment to make this clearer

Reply:
No, its TorchServe's production image. We are making use of docker tags to build different images vs having multiple Dockerfile. There is a comment already on L87